kolibri/core/tasks/decorators.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from functools import partial
+from importlib import import_module
+
+from django.apps import apps as django_apps
+
+from kolibri.core.tasks.job import JobRegistry
+from kolibri.core.tasks.job import Priority
+from kolibri.core.tasks.job import RegisteredJob
+from kolibri.core.tasks.utils import stringify_func
+
+
+logger = logging.getLogger(__name__)
+
+
+class _TaskDecorators(object):
```

Comment:
I think it would be good to just make `register` a top level module function, rather than using the class here.

Reply:
I personally like decorators of form `object.action` (that's why I left the class there :sweat_smile:). 

Anyway, since we only have one decorator let us get rid of class. 

In future if we think of a second decorator then we'll get back to class please! :laughing: